addons/web_editor/models/assets.py

```diff
@@ -29,7 +30,9 @@ def get_all_custom_attachments(self, urls):
         Returns:
             ir.attachment(): attachment records related to the given URLs.
         """
-        return self._get_custom_attachment(urls, op='in')
+        if not self.env.user.has_group('website.group_website_designer'):
```

Comment:
Can't use website in web_editor module, this will crash.
Need to be done in an override in website module.